### PR TITLE
add example

### PR DESCRIPTION
**Pull request by paulbastian**
Closes # 

## Preview Link

[click here for rendered preview of PR](https://github.com/<repo-name>/<branch-name>/<draft-name>.html)

### CODE REVIEW

File: draft-looker-oauth-attestation-based-client-authentication.md [hunk 1]
```diff
@@ -278,7 +278,7 @@ Implementers should be aware that using the same client attestation across multi
 
 The guidance provided by {{RFC7519}} and {{RFC8725}} applies.
 
-# IANA Considerations
+# Appendix AIANA Considerations
```
Comment (tplooker) [inline draft-looker-oauth-attestation-based-client-authentication.md:281]:
```suggestion
# Appendix A IANA Considerations
```

File: draft-looker-oauth-attestation-based-client-authentication.md [hunk 2]
```diff
@@ -299,6 +299,42 @@ This section registers the value "attest_jwt_client_auth" in the IANA "OAuth Tok
...
 
+# Additional Examples
+
+## Wallet Instance Attestation
+
+This non-normative example shows a client attestations used as an wallet instance attestation in the context of eIDAS 2.0, e.g. to secure a Type-1 configuration credential. The additional claims describe the wallet's device binding und user binding capabilities and the achievable level of assurance.
+
+~~~
+{
+	"typ": "vc+jwt",
```
Comment (paulbastian) [inline draft-looker-oauth-attestation-based-client-authentication.md:310]:
```suggestion
	"typ": "vc+sd-jwt",
```

File: draft-looker-oauth-attestation-based-client-authentication.md [hunk 3]
```diff
@@ -299,6 +299,42 @@ This section registers the value "attest_jwt_client_auth" in the IANA "OAuth Tok
...
+	"kid": "1"
+}
+.
+{
+	"iss": "https://attestation-service.ssi.tir.budru.de",
+	"sub" : "https://lissi.org",
+	"iat": 1541493724,
+	"exp": 1516247022,
+	"type": "WalletAttestation",
+	"wallet_name": "Lissi Dev",
```
Comment (paulbastian) [inline draft-looker-oauth-attestation-based-client-authentication.md:321]:
```suggestion
	"typ": "wallet-attestation+jwt",
	"alg": "ES256",
	"kid": "1"
}
.
{
	"iss": "https://attestation-service.ssi.tir.budru.de",
	"sub" : "https://lissi.org",
	"iat": 1541493724,
	"exp": 1516247022,
	"wallet_name": "Lissi Dev",
```

File: draft-looker-oauth-attestation-based-client-authentication.md [hunk 4]
```diff
@@ -299,6 +299,41 @@ This section registers the value "attest_jwt_client_auth" in the IANA "OAuth Tok
...
+
+~~~
+{
+	"typ": "wallet-attestation+jwt",
+	"alg": "ES256",
+	"kid": "1"
+}
+.
+{
+	"iss": "https://attestation-service.ssi.tir.budru.de",
```
Comment (tlodderstedt) [inline draft-looker-oauth-attestation-based-client-authentication.md:316]:
```suggestion
	"iss": "https://wallet.example.com",
```

File: draft-looker-oauth-attestation-based-client-authentication.md [hunk 5]
```diff
@@ -299,6 +299,41 @@ This section registers the value "attest_jwt_client_auth" in the IANA "OAuth Tok
...
+	"sub" : "https://lissi.org",
```
Comment (tlodderstedt) [inline draft-looker-oauth-attestation-based-client-authentication.md:317]:
```suggestion
	"sub" : "https://wallet.example.com"",
```

File: draft-looker-oauth-attestation-based-client-authentication.md [hunk 6]
```diff
@@ -299,6 +299,41 @@ This section registers the value "attest_jwt_client_auth" in the IANA "OAuth Tok
...
+	"iat": 1541493724,
+	"exp": 1516247022,
+	"wallet_name": "Lissi Dev",
+	"wallet_version": "1.6.0",
+	„key_type" : "STRONGBOX",
+	"user_authentication" : "APP_PIN_6_DIGITS",
+	"supported_LoA" : "https://eu-trust-list.eu/loa/high",
```
Comment (tlodderstedt) [inline draft-looker-oauth-attestation-based-client-authentication.md:324]:
```suggestion
```

File: draft-looker-oauth-attestation-based-client-authentication.md [hunk 7]
```diff
@@ -299,6 +299,41 @@ This section registers the value "attest_jwt_client_auth" in the IANA "OAuth Tok
 
 --- back
 
+# Additional Examples
+
+## Wallet Instance Attestation
+
+This non-normative example shows a client attestations used as an wallet instance attestation in the context of eIDAS 2.0, e.g. to secure a Type-1 configuration credential. The additional claims describe the wallet's device binding und user binding capabilities and the achievable level of assurance.
```
Comment (tlodderstedt) [inline draft-looker-oauth-attestation-based-client-authentication.md:306]: If you mention eIDAS terms, like "Type-1 configuration", you also need to add a reference to the ARF.

**Review by tlodderstedt — CHANGES_REQUESTED** [review]

File: draft-looker-oauth-attestation-based-client-authentication.md [hunk 8]
```diff
@@ -299,6 +299,41 @@ This section registers the value "attest_jwt_client_auth" in the IANA "OAuth Tok
...
+}
+.
+{
+	"iss": "https://attestation-service.ssi.tir.budru.de",
+	"sub" : "https://lissi.org",
+	"iat": 1541493724,
+	"exp": 1516247022,
+	"wallet_name": "Lissi Dev",
+	"wallet_version": "1.6.0",
+	„key_type" : "STRONGBOX",
```
Comment (peppelinux) [inline draft-looker-oauth-attestation-based-client-authentication.md:322]:
```suggestion
	"key_type" : "STRONGBOX",
```

File: draft-looker-oauth-attestation-based-client-authentication.md [hunk 9]
```diff
@@ -299,6 +299,41 @@ This section registers the value "attest_jwt_client_auth" in the IANA "OAuth Tok
...
+	"kid": "1"
+}
+.
+{
+	"iss": "https://attestation-service.ssi.tir.budru.de",
+	"sub" : "https://lissi.org",
+	"iat": 1541493724,
+	"exp": 1516247022,
+	"wallet_name": "Lissi Dev",
+	"wallet_version": "1.6.0",
```
Comment (peppelinux) [inline draft-looker-oauth-attestation-based-client-authentication.md:321]:
```suggestion
```

Why the attestation should disclose information like these?
Probably the User doesn't want to disclose the device name and the wallet version

**Review by peppelinux — CHANGES_REQUESTED** [review]

File: draft-looker-oauth-attestation-based-client-authentication.md (hunk 4 above)
Comment (paulbastian) [reply draft-looker-oauth-attestation-based-client-authentication.md:316]: I would rename this to attestation-service.com to signalize that these could be different?

File: draft-looker-oauth-attestation-based-client-authentication.md (hunk 9 above)
Comment (paulbastian) [reply draft-looker-oauth-attestation-based-client-authentication.md:321]: "wallet_name" is meant as a human readable name of the wallet solution, the technical identifier for a wallet solution would be registered as "sub"
"wallet_version" is the version of the wallet instance of the wallet solution. The certification and validity status of a wallet solution probably depends on the version, thus for lookup in a trust registry the version is useful

Comment (tlodderstedt) [reply draft-looker-oauth-attestation-based-client-authentication.md:321]: I see some value in the wallet name. wrt the version, as we discussed yesterday, I would assume whatever is needed to lookup data about the authorization of the wallet should be encoded in the iss or sub value.

File: draft-looker-oauth-attestation-based-client-authentication.md (hunk 4 above)
Comment (tlodderstedt) [reply draft-looker-oauth-attestation-based-client-authentication.md:316]: I would like to keep the example as simple as possible.  

I think this is a special case, which required explanation. If you provide text, I could incorporate into https://github.com/vcstuff/oid4vc-haip-sd-jwt-vc/pull/52

File: draft-looker-oauth-attestation-based-client-authentication.md (hunk 6 above)
Comment (paulbastian) [reply draft-looker-oauth-attestation-based-client-authentication.md]: added attested security context instead

File: draft-looker-oauth-attestation-based-client-authentication.md (hunk 9 above)
Comment (paulbastian) [reply draft-looker-oauth-attestation-based-client-authentication.md]: added version to sub

File: draft-looker-oauth-attestation-based-client-authentication.md [hunk 10]
```diff
@@ -299,6 +301,40 @@ This section registers the value "attest_jwt_client_auth" in the IANA "OAuth Tok
...
+	"kid": "1"
+}
+.
+{
+	"iss": "https://wallet-provider.com",
+	"sub": "https://wallet-provider.com/solution/wallet-1.6.0",
+	"iat": 1541493724,
+	"exp": 1516247022,
+	"wallet_name": "human readable wallet name",
+	"key_type" : "STRONGBOX",
```
Comment (paulbastian) [inline draft-looker-oauth-attestation-based-client-authentication.md:323]:
```suggestion
	"key_type" : "Strongbox",
```

**Comment by paulbastian** [comment]
@tlodderstedt please review

**Review by tplooker — APPROVED** [review]

**Pull request merged**